main.go

```diff
@@ -197,26 +205,49 @@ func (cli *Client) getUid() (string, error) {
 	return string(body[3:]), nil
 }
 
-func (cli *Client) GetBalance() (int, error) {
+func (cli *Client) GetBalance() (balance float64, currency string, err error) {
 	resp, err := cli.http.Get(urlDevices)
 	defer resp.Body.Close()
 
 	if err != nil {
-		return 0, err
+		return 0, "", err
 	}
 
 	body, _ := ioutil.ReadAll(resp.Body)
 	matches := reBalance.FindStringSubmatch(string(body))
 	if len(matches) == 0 {
-		return 0, errors.New("could not find balance data")
+		return 0, "", errors.New("could not find balance data")
+	}
+
+	balance, err = strconv.ParseFloat(strings.Replace(matches[1], ",", ".", -1), 64)
+	currency = matches[3]
+	if err != nil {
+		return 0, "", err
 	}
 
-	balance, err := strconv.Atoi(matches[1])
+	return balance, currency, nil
+}
+
+func (cli *Client) GetRemains() (string, error) {
+	resp, err := cli.http.Get(urlDevices)
+	defer resp.Body.Close()
+
 	if err != nil {
-		return 0, err
+		return "", err
+	}
+
+	body, _ := ioutil.ReadAll(resp.Body)
+	matches := reRemains.FindStringSubmatch(string(body))
+	if len(matches) == 0 {
+		return "", errors.New("could not find remains data")
+	}
+
+	remains := fmt.Sprintf("%s %s %s", matches[1], matches[2], matches[3])
+	if err != nil {
```

Comment:
There is no error here, no need for check.

Reply:
ok

Reply:
how can i fix it? new commit and PR?

Reply:
@bvp  just push new commit into this branch.